ui/packages/components/src/Modal/AlertModal.tsx

```diff
@@ -71,12 +80,17 @@ export function AlertModal({
                 {children}
                 <div className="flex justify-end gap-2 p-6 dark:border-slate-800">
                   <AlertDialog.Cancel asChild>
-                    <Button appearance="outlined" disabled={isLoading} label="No" />
+                    <Button
+                      appearance="outlined"
+                      disabled={isLoading}
+                      kind={cancelButtonKind}
```

Comment:
I don't think we'll need a cancelButtonKind! It's always a secondary Cancel action, so it's a basic outlined one!